flannel-image/flannel-image.kiwi.ini

```diff
@@ -1,28 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<!-- OBS-AddTag: _NAMESPACE_/flannel:%%LONG_VERSION%% _NAMESPACE_/flannel:%%LONG_VERSION%%-<RELEASE> -->
+<!-- OBS-ExcludeArch: i586 s390 -->
 
-<image schemaversion="6.5" name="_PRODUCT_-flannel-image">
+<image schemaversion="6.5" name="_PRODUCT_-flannel-image" xmlns:suse_label_helper="com.suse.label_helper">
   <description type="system">
     <author>SUSE Containers Team</author>
     <contact>containers@suse.com</contact>
-    <specification>Flannel running on a _DISTRO_ container guest</specification>
+    <specification>Network fabric for containers (flannel)</specification>
   </description>
   <preferences>
     <type
       image="docker"
       derived_from="obsrepositories:/_BASEIMAGE_">
       <containerconfig
         name="_NAMESPACE_/flannel"
-        tag="%%SHORT_VERSION%%"
+        tag="latest"
```

Comment:
Does it mean that the version will be not used as tag?

I'd suggest to use `additionaltags` parameter to include `%%SHORT_VERSION%%` and/or `%%LONG_VERSION%%` there.